storage/cloud-client/encryption_test.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2016 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import tempfile
+
+import encryption
+from gcloud import storage
+import pytest
```

Comment:
Should this match [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)?

Specifically, encryption should be in a third section if so.